sdk/core/azure-core/src/main/java/com/azure/core/credential/TokenRequestContext.java

```diff
@@ -13,6 +13,7 @@
  */
 public class TokenRequestContext {
     private final List<String> scopes;
+    private String claims;
```

Comment:
Would there be any thread safety concerns with claims?

Reply:
nope, a new instance of TRC is created per call.